include/CoreGraphics/CGImage.h

```diff
@@ -106,3 +86,8 @@ COREGRAPHICS_EXPORT bool CGImageIsMask(CGImageRef image) STUB_METHOD;
 
 COREGRAPHICS_EXPORT size_t CGImageGetHeight(CGImageRef image);
 COREGRAPHICS_EXPORT size_t CGImageGetWidth(CGImageRef image);
+
+COREGRAPHICS_EXPORT inline surfaceFormat _CGImageGetFormat(unsigned int bitsPerComponent,
```

Comment:
Good catch. I will put this function with the other helpers in CGSurfaceInfoInternal.h